examples/Example01_Basic_OneShot/Example01_Basic_OneShot.ino

```diff
@@ -22,16 +22,18 @@
   https://www.sparkfun.com/products/23518 - Qwiic Mini
 */
 
-#include "SparkFun_AS7331.h"
+#include <Arduino.h>
+#include <Wire.h>
+#include <SparkFun_AS7331.h>
 
 SfeAS7331ArdI2C myUVSensor;
 
-int8_t result = SFE_BUS_OK;
-
 void setup() {
   Serial.begin(115200);
   while(!Serial){delay(100);};
```

Comment:
Have you tested this on various platforms? I recall `while(!Serial)` doing weird things sometimes, but maybe that's outdated.

Also good to just test the whole library on other platforms to confirm everything is happy.

Reply:
Tested it on ESP32, nRF, and Atmel platforms. I've never had an issue with this tbh.

Reply:
Sounds good!